cfme/tests/configure/test_default_views_infra.py

```diff
@@ -21,13 +21,14 @@ def minimise_dict(item):
     else:
         return item
 
+# todo: infrastructure resource_pools were removed from here. need to move this test to
```

Comment:
Awesome notes here, this is what we want to do, I have ideas for rewriting the tests to take advantage of the new nav objects.